README.md

```diff
@@ -206,11 +213,14 @@ that you have a 32bit version of Python installed.
 
 ### Set up your django environment
 
+Before running any of the commands below, you should have all of the following running: couchdb, redis, elasticsearch,
+and kafka. To install and run kafka, follow the instructions here: 
```

Comment:
Ok, can we just wait til Ethan is online to clarify? I'm sure I'm mis-communicating something then, but I'm also sure that I didn't pull this out of thin air.